Dockerfile

```
@@ -458,12 +458,16 @@ RUN ./install_bioc.r \
     ids
 
 WORKDIR /home/rstudio/
-# AWS sCLI installation
+# AWS CLI installation
 RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
     unzip awscliv2.zip && \
     sudo ./aws/install && \
     rm -rf aws*
 
+# Install Desal latest release (v2.1.1)- converter for JSON, TOML, YAML, XML and CSV data formats
+RUN sudo wget -qO /usr/local/bin/dasel "https://github.com/TomWright/dasel/releases/latest/download/dasel_linux_amd64" && \
```

```suggestion
RUN sudo wget -qO /usr/local/bin/dasel "https://github.com/TomWright/dasel/releases/download/v2.1.1/dasel_darwin_amd64" && \
```

I actually meant this, so that we are always using this release

---

@jharenza,  updated.